PublicMapping/districtbuilder
# Display featured projects on Org screen

## Overview

Displays all of an Organization's featured projects in cards on the Organization screen, including thumbnail maps showing the outlines of all of the districts created within the project. 

- Move OrgProject type and refactor references
- Add featured projects cards
- format
- Clean up lint

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/111651015-3e237c80-87dc-11eb-9af3-33b541d7d1e7.png)

## Testing Instructions

- Ensure that you have a user, organization, and a project associated with the organization's template
- Navigate to the organization's admin page, and set the project as featured
- Go back to the main organization page - expect that district plans are displayed in cards underneath the organization's templates


Closes #565
